packages/skin/src/routes/_index/component/chip/+page.marko

```diff
@@ -26,10 +26,287 @@
+            <img
+                class="chip__media"
+                src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/dog_profile_optimized.jpg"
+                alt=""
+            >
+            <span class="chip__text">
+                Football
+            </span>
+        </button>
+
+        <a class="chip chip--expressive" href="https://www.ebay.com">
+            <img
+                class="chip__media"
+                src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/dog_profile_optimized.jpg"
+                alt=""
+            >
+            <span class="chip__text">
+                Football
+            </span>
+        </a>
+    </highlight-code>
+
+    <h3>Toggle expressive chip selected</h3>
+    <p>
+        For an link selected toggle chip, it should have a clipped selected text.
+    </p>
+    <div class="demo">
+        <div class="demo__inner">
+            <button
+                class="chip chip--expressive"
+                type="button"
+                aria-pressed="true"
+            >
+                <img
+                    class="chip__media"
+                    src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/dog_profile_optimized.jpg"
+                    alt=""
+                >
+                <span class="chip__text">
+                    Football
+                </span>
+            </button>
+            <span style="margin-right: 8px;"/>
+            <a
+                class="chip chip--selected chip--expressive"
+                href="https://www.ebay.com"
+            >
+                <img
+                    class="chip__media"
+                    src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/dog_profile_optimized.jpg"
+                    alt=""
+                >
+                <span class="chip__text">
+                    Football
+                    <span class="clipped">
+                        - filter applied
+                    </span>
+                </span>
+            </a>
+        </div>
+    </div>
+
+    <highlight-code type="html">
+        <button class="chip chip--expressive" type="button" aria-pressed="true">
+            <img
+                class="chip__media"
+                src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/dog_profile_optimized.jpg"
+                alt=""
+            >
+            <span class="chip__text">
+                Football
+            </span>
+        </button>
+
+        <a
+            class="chip chip--expressive chip--selected"
+            href="https://www.ebay.com"
+        >
+            <img
+                class="chip__media"
+                src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/dog_profile_optimized.jpg"
+                alt=""
+            >
+            <span class="chip__text">
+                Football
+                <span class="clipped">
+                    - selected
+                </span>
+            </span>
+        </a>
+    </highlight-code>
+
+    <h3 id="destructive-chip">
```

Comment:
Since destructive was original to chip and seems to be simpler than the others, can we move this one to the top, right after the static chip?